### PR TITLE
Clipsview track selection

### PR DESCRIPTION
**Pull request by grliszas14**
Resolves: https://github.com/audacity/audacity/issues/7516

### CODE REVIEW

File: src/projectscene/view/clipsview/selectionviewcontroller.cpp [hunk 1]
```diff
@@ -33,11 +80,23 @@ void SelectionViewController::onPositionChanged(double x, double y)
         return;
     }
 
+    Qt::KeyboardModifiers modifiers = QApplication::keyboardModifiers();
+
+    //! NOTE: always treat simultaneously pressed Ctrl and Shift as Ctrl
```
Comment (Eism) [inline src/projectscene/view/clipsview/selectionviewcontroller.cpp:85]: it looks like we can move it to a separate function and reuse

**Review by kryksyh — COMMENTED** [review]
Not sure what really causes this, but if the user shift+clicks on a track twice without moving on X, it resets playhead to 0

**Review by kryksyh — COMMENTED** [review]
Another minor _glitch_ , when the user ctrl+presses it resets the data selection, until the mouse is moved or released. Probably need to

```
    selectionController()->setDataSelectedStartTime(m_context->positionToTime(x1, true /*withSnap*/), false);
    selectionController()->setDataSelectedEndTime(m_context->positionToTime(x2, true /*withSnap*/), false);
```
    
 in onPrssed

File: src/projectscene/view/clipsview/selectionviewcontroller.cpp (hunk 1 above)
Comment (grliszas14) [reply src/projectscene/view/clipsview/selectionviewcontroller.cpp]: fixed 👍

**Review by Eism — APPROVED** [review]

File: src/projectscene/view/clipsview/clipslistmodel.cpp [hunk 2]
```diff
@@ -44,6 +44,9 @@ void ClipsListModel::init()
         Q_UNUSED(time);
         updateItemsMetrics();
     });
+    selectionController()->tracksSelected().onReceive(this, [this](const TrackIdList&) {
+        updateItemsMetrics();
```
Comment (Eism) [inline src/projectscene/view/clipsview/clipslistmodel.cpp:48]: I think here it is worth updating only those parameters that depend on the selection
am I right?

**Review by kryksyh — APPROVED** [review]

**Pull request merged**